packages/jest-util/src/add_instance_of_alias.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+export default function addInstanceOfAlias(target: Object, alias: Object) {
```

Comment:
I don't know if alias is a good name. I only know alias in terms of giving things a different name but this is setting a different constructor. How about `addInstanceOfObject` or `addInstanceOfConstructor` or `addInstanceParent` or similar?